src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,90 @@
+package Duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String taskType;
+
+    public Task(String description, String taskType) {
+        if (taskType.equalsIgnoreCase("event")) {
```

Comment:
This constructor class seems to be lengthy for the start. You could abstract away some details, while ensuring [SLAP](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard).